app/scripts/root.js

```diff
@@ -157,7 +157,9 @@ angular.module('openolitor-admin')
       };
 
       $scope.storeActiveLang = function(lang) {
-        $cookies.put('activeLang', lang);
+        var expireDate = new Date();
+        expireDate.setDate(expireDate.getDate() + 3650);
```

Comment:
Does this really work? In the console `new Date().getDate() + 3650` return 3680 (for today).

Reply:
It adds 3650 days to today as here:

![image](https://user-images.githubusercontent.com/13901369/113004676-133e0e80-9174-11eb-92af-0894ce7c6fe8.png)